scripts/run-hydra_cli-reference.sh

```diff
@@ -0,0 +1,53 @@
+#!/bin/bash
+# Exit script if you try to use an uninitialized variable.
+set -o nounset
+
+# Exit script if a statement returns a non-true return value.
+set -o errexit
+
+# Use the error status of the first failure, rather than that of the last item in a pipeline.
+set -o pipefail
+
+# The CLIDOC assignment is BASH specific i.e. not portable. Consider using this alternative assignment if $HOME does not expand properly
+# CLIDOC=~/"docs/docs/reference/hydra_cli-reference.md"
+# Add touch filename if .md does not exist
+
+CLIDOC="$HOME/docs/docs/reference/hydra_cli-reference.md"
```

Comment:
The `$HOME` reference is confusing here. `$HOME` is set to `/Users/<user>` on macOS and `/home/<user>` on Linux. The CLI script should work without any env vars set. You can cd into the project root relative to the bash script location regardless of the cwd using our standard [cd instruction for other scripts](https://github.com/ory/hydra/blob/master/scripts/run-genswag.sh#L5). You can then just do `CLIDOC=/docs/docs/reference/hydra_cli-reference.md`. Also we should stay consistent when it comes to spacing with `-` preferred as they're interpreted as spaces by search engines:

```suggestion
CLIDOC="./docs/docs/reference/cli.md"
```